### PR TITLE
Feature/bg 13

### PR DESCRIPTION
**Pull request by MaciekPon**
feat(settings): 
- add responsive styles for settings page

### CODE REVIEW

File: src/settings/settings.scss [hunk 1]
```diff
@@ -44,24 +55,47 @@ body {
...
+    font-size: $s-font-size-mobile;
+    margin-right: 0;
+    align-self: center;
+  }
 }
 
 .section__link--return:visited {
   color: $white-color;
-}
+}
```
Comment (ruljin) [inline src/settings/settings.scss:109]: Like we discussed on Discord - there is a bit to change in styles like missing font-weights, colors, paddings/margins, cursor on the button, etc.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/settings/settings.scss [hunk 2]
```diff
@@ -16,26 +16,37 @@ body {
   align-items: center;
   flex-direction: column;
   height: 100%;
-  width: 50%;
+  width: calc(100% - 30vw);
```
Comment (JustD1d1t) [inline src/settings/settings.scss]: Why using calc instead 70vw?

File: src/settings/settings.scss [hunk 3]
```diff
@@ -44,24 +55,47 @@ body {
...
   font-weight: $font-bold;
+
+  @media(max-width: $l-breakpoint) {
+    width: 350px;
+    height: 95px;
+  }
+
+  @media(max-width: $m-breakpoint) {
+    width: 225px;
+    height: 71px;
```
Comment (JustD1d1t) [inline src/settings/settings.scss:71]: You could use 70px :D

File: src/settings/settings.scss [hunk 4]
```diff
@@ -44,24 +55,47 @@ body {
...
-  margin: 10px;
   border: 0;
+  width: 150px;
   font-size: $m-font-size-desktop;
-  border-bottom: 1px solid $white-color;
+  border-bottom: 3px solid $white-color;
   color: $white-color;
+
+  @media(max-width: $m-breakpoint) {
+    width: 101px;
```
Comment (JustD1d1t) [inline src/settings/settings.scss:87]: same as up

**Review by ruljin — APPROVED** [review]

**Pull request merged**